dialogflow/api/Test/DialogflowSampleTests.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.Dialogflow.V2" Version="2.0.0" />
+    <PackageReference Include="Google.Cloud.Dialogflow.V2" Version="3.1.0" />
```

Comment:
Remove this package reference. You already get it when you reference the sample project.